src/Ombi/ClientApp/src/app/media-details/components/movie/panels/movie-information-panel.component.html

```diff
@@ -1,136 +1,148 @@
 <div *ngIf="movie" class="left-panel-details">
-    <div class="rating medium-font">
-        <span *ngIf="movie.voteAverage"
-            matTooltip="{{'MediaDetails.Votes' | translate }}  {{movie.voteCount | thousandShort: 1}}">
-            <img class="rating-small" src="{{baseUrl}}/images/tmdb-logo.svg"> {{movie.voteAverage | number:'1.0-1'}}/10
-        </span>
-        <span *ngIf="ratings?.critics_rating && ratings?.critics_score">
-            <img class="rating-small"
-                src="{{baseUrl}}/images/{{ratings.critics_rating === 'Rotten' ? 'rotten-rotten.svg' : 'rotten-fresh.svg'}}">
-            {{ratings.critics_score}}%
-        </span>
-        <span *ngIf="ratings?.audience_rating && ratings?.audience_score">
-            <img class="rating-small"
-                src="{{baseUrl}}/images/{{ratings.audience_rating === 'Upright' ? 'rotten-audience-fresh.svg' : 'rotten-audience-rotten.svg'}}">
-            {{ratings.audience_score}}%
-        </span>
-    </div>
-    <div *ngIf="streams?.length > 0" class="streaming-on">
-        <hr>
-        <span class="label">{{'MediaDetails.StreamingOn' | translate }}</span>
-        <div>
-        <span *ngFor="let stream of streams">
-            <img class="stream-small" [matTooltip]="stream.streamingProvider" src="https://image.tmdb.org/t/p/original{{stream.logo}}">
-        </span>
-        </div>
-    </div>
+  <div class="rating medium-font">
+    <span *ngIf="movie.voteAverage"
```

Comment:
do you mind sorting out the formatting as it's difficult to see the diff when the formatting of the file changes

Reply:
Sorry about that! Should be better now. Visual Studio's auto-formatting can be a pain 😩